packages/modals/src/elements/TooltipDialog/TooltipDialog.tsx

```diff
@@ -64,9 +72,10 @@ const TooltipDialogComponent = React.forwardRef<HTMLDivElement, ITooltipDialogPr
         restoreFocus: false
       });
 
-    const [floatingPlacement] = getFloatingPlacements(
+    const [floatingPlacement, fallbackPlacements] = getFloatingPlacements(
```

Comment:
if `props.fallbackPlacements` is `undefined`, `getFloatingPlacements` will generate opposite fallbackPlacements.

https://github.com/zendeskgarden/react-components/blob/0101f3ca87f49bfcc54864849b4c24e5d7f4fb10/packages/theming/src/utils/getFloatingPlacements.ts#L57-L77


Should we set a sensible default (e.g.: `['bottom']`) to both `Tooltip` and `TooltipDialog`?

Reply:
I wouldn't due to the fact that `floating-ui` already attempts to position within available space